### DISCUSSION TITLE
Is there a way to show lyrics for songs?

### DISCUSSION THREAD

**Discussion by ksharizard** [DeaDBeeF-Player/deadbeef#3048] [Q&A] [answered]
I have been trying to search for plugins for deadbeef which allow me to use my local .lrc files to show synced lyrics and/or online lyrics platform to show the lyrics. However, I have not been able to find any lyric plugin for deadbeef.

**Comment by kuba160** [comment, marked as answer]
There was some lyric plugin that was developed before but later it became incompatible. Recently somebody tried to resurrect it but it might not be 100% stable: https://github.com/AsVHEn/deadbeef-lyricbar

**Reply by AsVHEn** [reply]
> There was some lyric plugin that was developed before but later it became incompatible. Recently somebody tried to resurrect it but it might not be 100% stable: https://github.com/AsVHEn/deadbeef-lyricbar

No, it's not 100% stable, but I am almost getting something acceptable :rofl: .
